src/components/Header/Header.module.css

```diff
@@ -1,23 +1,30 @@
 .header {
-  height: 82px;
+  --header-height-sm: 3rem;
+  --header-height-lg: 5rem;
+}
+
+.header {
+  height: var(--header-height-sm);
   background-color: var(--black);
   box-shadow: 1px 1px 5px 0 var(--gray);
   position: fixed;
   display: flex;
   justify-content: space-between;
+  align-items: center;
```

Comment:
Amen for flexbox. No more guessing at padding / margin to center stuff vertically.